CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## To be released
 - Include pushclient module in android build
+- Include PushClient module in iOS build
```

Comment:
Can we make the casing consistent here with the `android` line above? (I'm not partial to which one, just that they're the same) :)